tests/config-keepassx-tests.h.cmake

```diff
@@ -5,4 +5,8 @@
 
 #define KEEPASSX_TEST_DATA_DIR "${KEEPASSX_TEST_DATA_DIR}"
 
+#cmakedefine WITH_XC_HTTP
```

Comment:
Are these defines used anywhere or is this just for when we have proper HTTP tests?